code/obj/item/organs/butt.dm

```diff
@@ -30,6 +30,7 @@
 	var/datum/organHolder/holder = null
 	var/sound/sound_fart = null // this is the life I live, making it so you can change the fart sound of your butt (that you can wear on your head) so that you can make artifact butts with weird farts
 	var/made_from = "butt"
+	var/robotic = 0 // ffs
```

Comment:
why are you adding a `robotic` var to butts?

Reply:
To bring it more in line with the rest of the organs. Without it, a cyber butts would be overwritten by custom mutant butts, and those custom mutant butts wouldn't be changed back to normal when un-mutating.

There's probably a better way for me to do this though.

Reply:
You can just istype() for cyverbutts, because you really shouldn't be doing `foo:robotic` etc., which would be the only way this is relevant...

Reply:
Okay, I reverted the changes I did to butts, and did the thing you said. It actually seems to work now!

I also noticed that humans werent always getting their limbs colorized properly on-spawn, as well as humans spawned through the spawn command. I fixed this.